patch_api/rest.py

```diff
@@ -208,8 +208,8 @@ def request(
         timeout = None
         if _request_timeout:
             if isinstance(
-                _request_timeout, (int,) if six.PY3 else (int, long)
-            ):  # noqa: E501,F821
+                _request_timeout, (int,) if six.PY3 else (int, int)
+            ):  # noqa: E501
```

Comment:
This fixes the `flake8` warning (and has been updated in the template). `long` has been deprecated in python 3